sycl/unittests/program_manager/EliminatedArgMask.cpp

```diff
@@ -184,8 +184,7 @@ sycl::detail::ProgramManager::KernelArgMask getKernelArgMaskFromBundle(
               !ExecKernel->MSyclKernel->isCreatedFromSource());
 
   return sycl::detail::ProgramManager::getInstance().getEliminatedKernelArgMask(
-      ExecKernel->MOSModuleHandle, ContextImpl, DeviceImpl, Program,
```

Comment:
Thank you, Dmitry, for clearing unused variables. Could you please then remove definition of ContextImpl and DeviceImpl at the beginning of this function.

Reply:
@mlychkov, @dm-vodopyanov, please, do this clean-up in follow-up patches.